script.js

```diff
@@ -0,0 +1,39 @@
+var numberStart;
+var numberEnd;
+var jump;
+
+// get value from input tag when user fill in and convert value to Float
+function getNumberStart() {
+  numberStart = parseFloat(document.getElementById("numberstart").value);
+};
+
+function getNumberEnd() {
+  numberEnd = parseFloat(document.getElementById("numberend").value);
+}
+
+jump = 1;
+function getJump() {
+    jump = parseFloat(document.getElementById("jump").value);
+}
+
+// genarate number list
+function genNumberList() {
+  var numberList = "";
+  for (var index = numberStart; index <= numberEnd; index += jump) {

```

Comment:
Thiếu case thì phải, ví dụ nếu user xoá số ở một trong các input thì c hiểu giá trị đó sẽ là undefined và nó cũng sẽ trigger change event, nhưng nếu logic như vậy thì e vẫn sẽ chạy vòng for này?